app/ui/src/polyfills.ts

```diff
@@ -58,15 +58,3 @@ import 'zone.js/dist/zone';  // Included with Angular CLI.
 /***************************************************************************************************
  * APPLICATION IMPORTS
  */
-
-import 'rxjs/add/operator/do';
-import 'rxjs/add/observable/empty';
-import 'rxjs/add/observable/from';
-import 'rxjs/add/observable/merge';
-import 'rxjs/add/observable/of';
-import 'rxjs/add/operator/filter';
-import 'rxjs/add/operator/first';
-import 'rxjs/add/operator/map';
-import 'rxjs/add/operator/mergeMap';
-import 'rxjs/add/operator/switchMap';
-import 'rxjs/add/operator/take';
```

Comment:
This is the only part that strikes me a bit. In fairness the location of the imports does not matter that much as long as those are imported upfront, prior to any bits and pieces of the actual project. However, I would have strived to keep these imports at `polyfills.ts` in order not to clutter more an already cluttered workspace such as `app/ui/src`. Although these imports are not strictly _polyfills_, they address the lack of an actual native functional method by extending the `Observable` prototype, so those can be considered as such.

Ultimately this is a matter of preference and the whole point is to keep the amount of files we have at the top in check.

Reply:
Yeah, let's see what happens if I put them in `polyfills.ts` should work in theory.